src/app/code/community/Omise/Gateway/controllers/Callback/ValidateoffsitecitipointsController.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
The same goes for this

Reply:
same here @jonrandy let's focus on current architecture, and not request changes that requires some refactoring.

Reply:
See above